### PR TITLE
A couple of typos

### PR DESCRIPTION
**Pull request by mavroudisv**

### CODE REVIEW

File: _posts/2016-9-26-ml-theory-pt2.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ If you noticed, all our analysis up till now was focusing on a **single** hypoth
 
  In order for the entire hypothesis space to have a generalization gap bigger than $\epsilon$, at least one of its hypothesis: $h_1$ **or** $h_2$ **or** $h_3$ **or** ... etc should have. This can be expressed formally by stating that:
 
- $$\mathbb{P}\left[\sup_{h \in \mathcal{H}}|R(h) - R_\text{emp}(h)| > \epsilon\right] = \mathbb{P}\left[\bigcup_{h \in \mathcal{H}} |R(h) - R_\text{emp}(h)| > \epsilon\right]$$
+ $$\mathbb{P}\left[\sup_{h \in \mathcal{H}}|R(h) - R_\text{emp}(h)| > \epsilon\right] \leq \mathbb{P}\left[\bigcup_{h \in \mathcal{H}} |R(h) - R_\text{emp}(h)| > \epsilon\right]$$
```
Comment (Mostafa-Samir) [inline _posts/2016-9-26-ml-theory-pt2.md]: The probability of the whole space having a generalization gap bigger than epsilon _**is**_ the probability of at least one of the hypotheses in the space having such gap. So the equal sign here is correct, the less-than-or-equal appears in the next equation when we bound the probability of the union by the sum of its individual probabilities.

**Review by Mostafa-Samir — CHANGES_REQUESTED** [review]
Thanks @mavroudisv for the fixes. One change (in the below comment) and the PR will be ready to be merged.

**Comment by mavroudisv** [comment]
I believe my latest commit fixes the mistake I introduced. Thanks for the neat intro to ML Theory. :)

**Comment by Mostafa-Samir** [comment]
You're welcome @mavroudisv! Thank you for the typo fix :+1: Merging now.

**Pull request merged**